### PR TITLE
Fix http.ex using module attributes for values not known at compile

### LINKED ISSUE #1: nil module attributes in http.ex
Looks like the module attributes (@token, @time and @amount) are not known at compile time so get set to nil. 

Submitting a PR to fix this.

### PR DESCRIPTION
**Pull request by hankditton**
Fixes #1

### CODE REVIEW

File: lib/godfist/http.ex [hunk 1]
```diff
@@ -30,16 +27,16 @@ defmodule Godfist.HTTP do
     url = Map.get(@endpoint, region)
 
     region
-    |> ExRated.check_rate(@time, @amount)
+    |> ExRated.check_rate(time, amount)
```
Comment (aguxez) [inline lib/godfist/http.ex]: `time` and `amount` are function names.

File: lib/godfist/http.ex [hunk 2]
```diff
@@ -30,16 +27,16 @@ defmodule Godfist.HTTP do
...
     |> parse(url, rest)
   end
 
   defp parse({:ok, _}, url, rest) do
     case String.contains?(rest, "?") do
       true ->
-        get_body("#{url <> rest}&api_key=#{@token}")
+        get_body("#{url <> rest}&api_key=#{token}")
```
Comment (aguxez) [inline lib/godfist/http.ex]: `token` is a function name

File: lib/godfist/http.ex [hunk 3]
```diff
@@ -30,16 +27,16 @@ defmodule Godfist.HTTP do
...
       _ ->
-        get_body("#{url <> rest}?api_key=#{@token}")
+        get_body("#{url <> rest}?api_key=#{token}")
```
Comment (aguxez) [inline lib/godfist/http.ex]: `token` is a function name

**Review by aguxez — CHANGES_REQUESTED** [review]
Thank you for the PR!, however, before this gets merged I'd like to propose some changes to erase some warnings regarding new function names without parentheses.

**Review by aguxez — APPROVED** [review]

**Comment by hankditton** [comment]
Sorry, total elixir noob here. Thanks for catching that.

**Pull request merged**